### PR TITLE
[BEAM-14013] Add PreCommit Kotlin examples Jenkins Job

### PR DESCRIPTION
**Pull request by benWize**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by benWize** [comment]
Run Kotlin_Examples PreCommit

**Comment by benWize** [comment]
Results of new Jenkins Job for Kotlin examples: https://ci-beam.apache.org/job/beam_PreCommit_Kotlin_Examples_Phrase/1/

**Comment by benWize** [comment]
R: @kileys

File: .test-infra/jenkins/job_PreCommit_Kotlin_Examples.groovy [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import PrecommitJobBuilder
+
+PrecommitJobBuilder builder = new PrecommitJobBuilder(
+    scope: this,
+    nameBase: 'Kotlin_Examples',
+    gradleTask: ':examples:kotlin:preCommit',
+    triggerPathPatterns: [
+      '^model/.*$',
+      '^sdks/java/.*$',
+      '^runners/google-cloud-dataflow-java/.*$',
```
Comment (kileys) [inline .test-infra/jenkins/job_PreCommit_Kotlin_Examples.groovy:28]: The current precommit only runs on Spark, Flink and direct runners. Let's either split it out to trigger for those runners

Comment (benWize) [reply .test-infra/jenkins/job_PreCommit_Kotlin_Examples.groovy]: Ok, changed!

**Comment by kileys** [comment]
LGTM

**Pull request merged**